app/controllers/documents_controller.rb

```diff
@@ -99,6 +101,16 @@ def validate_step
 
 private
 
+  def audit(activity_type, audit_comment = nil, *activity_information)
```

Comment:
I like this method, but I don't think a splat is what you're looking for here for activity_information. I'd just use `activity_information = nil` like you have with audit_comment

Reply:
in fact, I think keyword arguments might be an even cleaner way of doing this: https://thoughtbot.com/blog/ruby-2-keyword-arguments

Reply:
I gave this a go with 
```
  def audit(activity_type, audit_comment:, activity_information:)
  ```
  
  also tried assigning to 
  
  ```  def audit(activity_type, audit_comment: nil, activity_information: nil)```
  
  and ```  def audit(activity_type, audit_comment: "nil", activity_information: "nil") ```
  
  but could not achieve the same effect that we have with the current method, so I'll keep it - open to suggestions on how it could actually be done better but I couldn't see a way to use keyword arguments with default to nil, so the current method is looking like:
  
  
  ```   def audit(activity_type, audit_comment = nil, activity_information = nil)
    Audit.create!(
      planning_application_id: @planning_application.id,
      user: current_user,
      audit_comment: audit_comment,
      activity_information: activity_information,
      activity_type: activity_type,
    )
  end
  ```